{% extends "base.html" %}
{% block home %}

<head>
<title>Computational Anatomy Blog</title>
<link href="/static/css/blog/style.css" rel="stylesheet">
<link href="/static/css/blog/bootstrap.css" rel="stylesheet" media="screen">

<!---- start-smoth-scrolling---->
<script type="text/javascript" src="/static/js/blog/move-top.js"></script>
<script type="text/javascript" src="/static/js/blog/easing.js"></script>
<script type="text/javascript">
			jQuery(document).ready(function($) {
				$(".scroll").click(function(event){		
					event.preventDefault();
					$('html,body').animate({scrollTop:$(this.hash).offset().top},1000);
				});
			});
		</script>
<!--start-smoth-scrolling-->
</head>
	<div class="single">
		<div class="container">
				<div class="single-top">
						<a href="#"><img class="img-responsive" src="/static/img/blog/Method-April17.png" alt=" " style="height:250%; width:250%;margin-left:1%;"></a>
					<div class=" single-grid">
						<h4>POOLED CRISPR SCREENING WITH SINGLE-CELL TRANSCRIPTOME READOUT</h4>				
							<ul class="blog-ic">
								<li><a href="#"><span> <i  class="glyphicon glyphicon-user"> </i>Adam Cribbs</span> </a> </li>
		  						 <li><span><i class="glyphicon glyphicon-time"> </i>April 15, 2017</span></li>	
		  						 <!-- Need to change the icons to font awsome ones -->	  						 	
		  					</ul>
<div class="single-blog">	  						

<p>I decided to pick this paper <b style=color:black;>“Pooled CRISPR screening with single-cell transcriptome readout”</b> published in Nature Methods in Jan 2017 because I am interested in using CRISPR screening in one of the projects. It’s not the first publication using single cell transcriptomics with a CRISPR screen but it is the most recent and well worth a look. I will attempt to summarise it further here:</p>

<p>Pooled CRISPR knockout screening is a powerful technique for interrogating the biological function of genes in a non-biased manner. However, this type of screening is limited to identification of genes that influence cellular growth or viability phenotypes. More recently, a number of new methods that combine CRISPR screening with single cell RNA-seq analysis have linked the effect of knocking out a gene on the level of other transcribed genes (<b style=color:black;>Dixit, A. et al. Cell 2016, Jaitin, D.A. et al. Cell 2016, Adamson, B. et al. Cell 2016</b>). </p>

<p>.In <b style=color:black;>Datlinger et al 2017</b>, the authors refine the technique and combine it into an easy to follow workflow. While there is nothing really of significant novel value to this paper it is easy to read and should provide adequate depth of writing to be able to adapt and follow in another laboratory. However, like all of these slightly novel nature methods papers, I think it would have been so much more informative and useful if it were written up into a nature protocols paper.</p>

<p>Personally speaking, I have performed a number of whole genome screens in the past. In my early attempts, before genome sequencing, I used a plate-based screen. However this is manually laborious, time consuming and does not always give reproducible results. More recently, I have performed a number of pooled shRNA screen and have had good experiences with them. However, I know that they can be difficult to set up in the beginning, however once running they are very easy to perform and computationally easy analyse. The key to all of these techniques is having good n numbers and good validation using single knockdowns, preferably using a number of different assays. I can see the huge potential for these single cell/CRISPR screens, and I plan on using them in my future work, however having experience with both single cell and pooled shRNA screens I know its never as straight forward as hoped. </p>


</div>						
</div>
</div>	

	
	


{% endblock %}