<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Science</title>

<script src="site_libs/header-attrs-2.1/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 66px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h2 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h3 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h4 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h5 {
  padding-top: 71px;
  margin-top: -71px;
}
.section h6 {
  padding-top: 71px;
  margin-top: -71px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Adam P Cribbs</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="science.html">Science</a>
</li>
<li>
  <a href="labnews.html">Lab News</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="contact.html">Contact</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Code
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="https://acribbs.github.io/siteymlgen/index.html">
        <span class="fa fa-archive"></span>
         
        siteymlgen
      </a>
    </li>
    <li>
      <a href="http://single-cell.readthedocs.io">
        <span class="fa fa-archive"></span>
         
        scflow
      </a>
    </li>
    <li>
      <a href="https://github.com/Acribbs/tRNAnalysis">
        <span class="fa fa-archive"></span>
         
        tRNAnalysis
      </a>
    </li>
    <li>
      <a href="https://github.com/cgat-developers">
        <span class="fa fa-archive"></span>
         
        cgat-developers
      </a>
    </li>
    <li>
      <a href="https://github.com/Acribbs/deseq2_report">
        <span class="fa fa-archive"></span>
         
        Deseq2 report
      </a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:adam.cribbs@dnorms.ox.ac.uk">
    <span class="fa fa-envelope-o"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/Acribbs">
    <span class="fa fa-github"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/CribbsP">
    <span class="fa fa-twitter"></span>
     
  </a>
</li>
<li>
  <a href="https://www.ndorms.ox.ac.uk/team/adam-cribbs">
    <span class="fa fa-globe-europe"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Science</h1>

</div>


<p><link href="static/css/custom.css" rel="stylesheet" ></p>
<p><img src="static/img/science.png" /></p>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/CZI.png" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
The Tendon Seed Network
</h2>
<h3>
Human Cell Atlas
</h3>
<h4>
The Botnar Research Centre, University of Glasgow, University of Copenhagen and University of Zurich
</h4>
<p>
<a href="https://chanzuckerberg.com/science/programs-resources/humancellatlas/seednetworks/the-tendon-seed-network/">The Tendon Seed Network</a> will spatially define the transcriptome of extracellular matrix-rich tissues such as tendon across multiple anatomic and micro-anatomic sites. To enable this, the project will develop clinical, laboratory, bioinformatics, and mathematical modelling tools and platforms.
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/maxresdefault.jpg" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
Hypoxia and brest cancer
</h2>
<h3>
Prof Adrian Harris
</h3>
<h4>
Weatherall Institute of Molecular Medicine, University of Oxford
</h4>
<p>
I am leading the computational analysis of single-cell data to identify breast cancer responses to hypoxia.
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/bayer.png" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
Research Alliance on Women’s Health
</h2>
<h3>
Oxford Bayer Healthcare Collaborative
</h3>
<h4>
Weatherall Institute of Molecular Medicine, University of Oxford
</h4>
<p>
Leading the integration of multi-omic data to identify drug targets for uterine fibroids and PCOS.
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/medulloblastoma.jpg" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
Understanding the epigenetic mechanisms of chemotherapy resistance in medulloblastoma
</h2>
<h3>
Dr. Tuğba Bağcı Önder
</h3>
<h4>
Koç University School of Medicine, Turkey
</h4>
<p>
Recently, Tugba and I were awarded a Royal Society Advanced Newton award to identify epigenetic drug targets for the reversal of chemotherapy resistance in medulloblastoma.
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/F2large.jpg" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
Multi-omic approach to identify new biomarkers in preeclampsia patients
</h2>
<h3>
Dr Manu Vatish
</h3>
<h4>
Nuffield Department of Obstetrics and Gynaecology, University of Oxford
</h4>
<p>
I am working in collaboration with Prof Ian Sargent and Dr Manu Vatish at the University of Oxford (Gynaecology). We are using proteomics and transcriptomics to identify biomarkers that could be useful in the diagnostic of preeclampsia. Our focus is directed towards investigating the role of extracellular vesicles, which are secreted in large volumes during preeclampsia. Extracellular vesicles persist in the plasma of patients for extended periods, which makes them an attractive biomarker. The data have revealed a number of proteins, mRNA and miRNA species that are differentially expressed between the placenta tissue and placental derived extracellular vesicles in normal and preeclampsia patients. The next phase of the project is to validate these potential biomarkers in a large number of patients. I plan to employ machine-learning approaches to develop a diagnostic tool for accurately assessing patient disease development, ultimately leading to better treatment outcomes.
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/AS.jpeg" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
Epigenetics of CD4 T cells in ankylosing spondylitis
</h2>
<h3>
Dr Liye Chen and Prof Paul Bowness
</h3>
<h4>
Botnar Research Centre, University of Oxford
</h4>
<p>
In this collaboration I am leading the computational analysis of a number of projects focused on the epigenetics of T cells in the autoimmune disease AS.
</p>
</div>
</div>
<br /><br />
<div class="cv3-envaleur">
<h2 style="text-align:center;font-family:Helvetica; font-color:black;font-size: 35px;">
<span>Past projects</span>
</h2>
<p><br /></p>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/metabolism.jpg" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
Metabolism of T helper cells
</h2>
<h3>
Prof Udo Oppermann and Prof Marc Feldmann
</h3>
<h4>
Botnar Research Centre, University of Oxford
</h4>
<p>
In this project I have been investigating how inhibiting specific epigenetic pathways can lead to altered metabolic programming.
</p>
<p>
This work was published as <a href="https://www.pnas.org/content/117/11/6056">Histone H3K27me3 demethylases regulate human Th17 cell development and effector functions by impacting on metabolism</a>
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/NKcells.jpg" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
Understanding the epigenetic pathways involved in Natural killer cell activation
</h2>
<h3>
Prof Udo Oppermann and Prof Sir Marc Feldmann
</h3>
<h4>
Botnar Research Centre, University of Oxford
</h4>
<p>
In this project I undertook the wet-lab and computational work that led to identification of a number of epigenetic pathways that control inflammatory function. This project was initiated during my postdoctoral work in the Oppermann and Feldmann laboratories.
</p>
<p>
This work was published as <a href="https://www.ncbi.nlm.nih.gov/pubmed/29301935">Inhibition of histone H3K27 demethylases selectively modulates inflammatory phenotypes of natural killer cells.</a>
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/nri1713-f2.jpg" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
B cell development in autoimmunity
</h2>
<h3>
Prof Richard Cornall and Dr Consuelo Anzilotti
</h3>
<h4>
WIMM, University of Oxford
</h4>
<p>
I led the computational analysis of the transcriptome data of a novel B cell development autoimmune disease.
</p>
<p>
This work was published as <a href="https://www.imm.ox.ac.uk/publications/950562">An essential role for the Zn2+ transporter ZIP7 in B cell development </a>
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/Dupuytren.jpg" style="width:60%;height:60%;"></p>
</div>
<div class="project-right">
<h2>
Investigating BET bromodomains in Dupytren’s disease
</h2>
<h3>
Dr Fiona McCann and Dr Lynn Williams
</h3>
<h4>
Kennedy Institute of Rheumatology, University of oxford
</h4>
<p>
I am working in collaboration with Dr Lynn Williams and Dr Fiona McCann at the Kennedy Institute of Rheumatology to analyse transcriptome data from patients with Dupytren’s disease.
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/nri3384-f3.jpg" style="width:60%;height:60%;"></p>
</div>
<div class="project-right">
<h2>
Single cell analysis of gamma delta T cells within salmonella infected patients
</h2>
<h3>
Prof Alison Simmons
</h3>
<h4>
WIMM, University of Oxford
</h4>
<p>
This project is a collaboration with Prof Alison Simmons, University of Oxford (WIMM), using single cell transcriptomic approaches to identify T cell heterogeneity following salmonella infection in healthy individuals. This project has allowed me to apply multiple statistical approaches to model T cell heterogeneity using a combination of both Frequentist and Bayesian statistical approaches.
</p>
</div>
</div>
<div id="project-3">
<div class="project-left">
<p><img src="static/img/1200x900f.jpg" style="width:70%;height:70%;border-radius: 5px"></p>
</div>
<div class="project-right">
<h2>
Designing CRISPR Cas gRNA across splice junctions in the human genome
</h2>
<h3>
Dr Daniel Ebner and Dr Andrew Bassett
</h3>
<h4>
Target Drug Discovery Institute, University of Oxford
</h4>
<p>
This was the first coding project that I was involved with during my training at CGAT. I established a computational infrastructure and website to identify and rank the effectiveness of designing gRNAs to splice regions in proteins across the genome. Although successful, this project didn’t lead to a publication because a rival published before us, however it did allow me to develop my skills in SQL and website development.
</p>
</div>
</div>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
